e2e/exhaustive_suite_test.go

```diff
@@ -55,7 +55,7 @@ func generateWorkloads([]workloads.Workload) {
 			Path:     GITPATH,
 			Revision: GITREVISION,
 			AppName:  APPNAME,
-			Name:     fmt.Sprintf("Deploy-%s", suffix),
+			Name:     fmt.Sprintf("Deployment-%s", suffix),
```

Comment:
Deploy is indeed confusing. I think the a better way is to remove the "Deploy-" prefix.